functions/Save-DbaDiagnosticQueryScript.ps1

```diff
@@ -72,16 +72,17 @@ function Save-DbaDiagnosticQueryScript {
     $glenberrysql = @()
     $RssPostFilter = "SQL Server Diagnostic Information Queries for*"
     $DropboxLinkFilter = "*dropbox.com*"
-    $LinkTitleFilter = "*Diagnostic*"
+    $LinkTitleFilter = "*Diagnostic Information Queries*"
+    $ExcludeSpreadsheet = "*BlankResutsSpreadsheet*"
 
     foreach ($post in $Feed.item) {
         if ($post.title -like $RssPostFilter) {
             # We found the first post that matches it, lets go visit and scrape.
             $page = Get-WebData -uri $post.link
-            $glenberrysql += ($page.Links | Where-Object { $_.href -like $DropboxLinkFilter -and $_.innerText -like $LinkTitleFilter } | ForEach-Object {
+            $glenberrysql += ($page.Links | Where-Object { $_.href -like $DropboxLinkFilter -and $_.outerHTML -like $LinkTitleFilter -and $_.outerHTML -notlike $ExcludeSpreadsheet } | ForEach-Object {
```

Comment:
Why did this have to change got both Windows and Linux? It worked previously on Windows didn't it?

Reply:
It did, I was making to the same code worked on both platforms without having to get into writing `if/else`. The previous code also overlooked the blank Excel files that I believe he is now including. This is something that didn't work in Core on windows as well. So it is more of a Core vs WindowsPS issue.

Reply:
Yeah I think the iwr changed in core...